frontend/src/views/producer/dashboard/event-financial-panel/EventFinancialPanel.jsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react';
+import styled, { withTheme } from 'styled-components';
+import Typography from '../../../../components/typography';
+import Icon from 'antd/lib/icon';
+import { connect } from '../../../../utils';
+import { values } from 'mobx';
+
+const Wrapper = styled.div`
+  margin: 1rem;
+  flex-basis: 20rem;
+  margin-top: calc(3rem + 5rem);
+  background-color: white;
+  border-radius: 8px;
+  overflow: auto;
+  padding: 1rem;
+  ${props => !props.show && 'display: none;'};
+`;
+const StyledList = styled.li`
+  list-style: none;
+  ul {
+    padding: 0.5rem;
+  }
+  i {
+    color: ${props => props.color};
+    margin-right: 0.5rem;
+    font-weight: 800;
+  }
+`;
+
+export const ListItem = ({ style, iconName, title, content, color }) => (
```

Comment:
Instead of the `content`prop, could you use `children` for the content? It would result in a nicer markup, when using this component.

Reply:
yes, i will do that